code/obj/machinery/door/airlock.dm

```diff
@@ -219,6 +219,19 @@ Airlock index -> wire color are { 9, 4, 6, 7, 5, 8, 1, 2, 3 }.
 		. = ..()
 		STOP_TRACKING
 
+	get_desc()
+		var/healthpercent = src.health/src.health_max * 100
+		switch(healthpercent)
+			if(90 to 99) //dont want to clog up the description unless it's actually damaged
+				. += "It seems to be in mostly good condition"
+			if(75 to 89)
+				. += "[src] seems slightly [pick("dinged up", "dented", "damaged", "scratched")]"
```

Comment:
since these are stuck on the end of the description it doesn't make sense to add `[src]` again here since that produces `This is a window.
A window. The window is barely intact.`